Framework/Geometry/test/ComponentInfoBankHelpersTest.h

```diff
@@ -72,101 +72,81 @@ class ComponentInfoBankHelpersTest : public CxxTest::TestSuite {
     TS_ASSERT(!isDetectorFixedInBank(componentInfo, componentInfo.root()));
   }
 
-  void test_isAnyBank_false_for_tubes() {
-
+  void test_isSaveableBank_false_for_tubes() {
+    // test instrument with detector tubes to test that IsSaveableBank will
+    // return false detector tubes
```

Comment:
Also needs comments about the parameters. What do `2` and `2` represent. better to have `(2 /\*some comment\*/, 2 /\*some comment\*/)'

Reply:
done